alibi_detect/utils/pytorch/tests/test_kernels_pt.py

```diff
@@ -3,7 +3,8 @@
 import pytest
 import torch
 from torch import nn
-from alibi_detect.utils.pytorch import GaussianRBF, DeepKernel
+from typing import Union
+from alibi_detect.utils.pytorch import GaussianRBF, DeepKernel, BaseKernel, RationalQuadratic, Periodic
 
```

Comment:
Missing explicit test for `infer_kernel_parameter`

Reply:
Will add.